data/tsv/bre_latn_phonemic.tsv

```diff
@@ -23,8 +23,6 @@ aour	ɔ w r
 arall	a r a lː
 aramaeg	a r a m a ɛ k
 arar	a ʁ a ʁ
-arc'hanti	a ʁ χ ɑ̃ n t i
```

Comment:
I don't know anything about Breton spelling but it looks like c'h is just how you spell dorsal fricatives of various types? These should probably not be filtered.